pkgs/by-name/lm/lm_sensors/package.nix

```diff
@@ -80,28 +63,24 @@ stdenv.mkDerivation {
     "MANDIR=${placeholder "man"}/share/man"
     # This is a dependency of the library.
     "ETCDIR=${placeholder "out"}/etc"
+    "BUILD_SHARED_LIB=${if stdenv.hostPlatform.isStatic then "0" else "1"}"
+    "BUILD_STATIC_LIB=${if stdenv.hostPlatform.isStatic then "1" else "0"}"
```

Comment:
Does `${toString stdenv.hostPlatform.isStatic}` work? The only difference is that `toString false == ""`. Not sure if it breaks stuff.1